ui/helpers/url.js

```diff
@@ -20,6 +20,22 @@ export const createQueryParams = function createQueryParams(params) {
   return `?${query.toString()}`;
 };
 
+// For API calls, we want arrays to be ``id=1&id=2&id=3`` not ``id=1,2,3``
+// Because the python ``getlist`` function on parameters can't handle the latter.
+export const createApiQueryParams = function createQueryParams(params) {
```

Comment:
Might want to consider using the `query-string` library here (could probably use for other things too): https://www.npmjs.com/package/query-string

Reply:
Thanks for the tip!  I'll checkout out ``query-string``.  I'm curious how it deals with multiple values for the same "field".  :)